lgc/patch/PatchPreparePipelineAbi.cpp

```diff
@@ -121,7 +121,6 @@ bool PatchPreparePipelineAbi::runImpl(Module &module, PipelineShadersResult &pip
   return true; // Modified the module.
 }
 
-// =====================================================================================================================
```

Comment:
Is this removed by accident?

Reply:
Yes, it is removed by accident, will change it back, man thanks~